### PR TITLE
Add `vesc-comm` to WIP drivers

### PR DESCRIPTION
**Pull request by chocol4te**

### CODE REVIEW

**Comment by ryankurte** [comment]
hey, thanks for the PR! what is a VESC (and could it be included in the description?

**Comment by chocol4te** [comment]
Ah sorry, a VESC is a motor driver for light electric vehicles, drones and robotics. https://vesc-project.com/Hardware. They have their own communication protocol which I have started implementing, but the protocol's sole use is for these devices, so I wasn't sure what else to call it.

**Comment by ryankurte** [comment]
ooh interesting, and fair enough / it's a good name.

File: README.md [hunk 1]
```diff
@@ -447,6 +447,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 - [MCP49xx] - SPI - 8/10/12-bit DACs like MCP4921, MCP4922, MCP4801, etc. - ![crates.io](https://img.shields.io/crates/v/mcp49xx.svg)
 - [DW1000] - SPI - Radio transceiver for communication based on IEEE 802.15.4 and position tracking - ![crates.io](https://img.shields.io/crates/v/dw1000.svg)
 - [radio-at86rf212] - SPI - Sub GHz 802.15.4 radio transceiver ![crates.io](https://img.shields.io/crates/v/radio-at86rf212.svg)
+- [vesc-comm] - A driver for communicating with VESCs ![crates.io](https://img.shields.io/crates/v/vesc-comm.svg)
```
Comment (ryankurte) [inline README.md]: could this contain something like `VESC-Project Electronic Speed Controller` to be more clear as to what VESCs are?

Comment (chocol4te) [reply README.md]: Done :)

**Pull request merged**